.github/workflows/ci.yml

```diff
@@ -1,20 +1,36 @@
-name: Project Tests
+name: Continuous Integration
 on:
   push:
     branches:
-      - master
       - develop
+      - master
   pull_request:
     branches:
-      - master
       - develop
+      - master
```

Comment:
Have tried to rename the job (build-test-push) to be more accurate, but this causes GitHub to think there are two jobs (build-test-push & test), this prevents the PR from passing checks, and so cannot be merged - but minor detail